lib/logstash/plugin_mixins/jdbc/value_tracking.rb

```diff
@@ -62,7 +62,7 @@ def common_set_initial(method_symbol, default)
 
   class NumericValueTracker < ValueTracking
     def set_initial
-      common_set_initial(:gcd, 0)
+      common_set_initial(:abs, 0)
```

Comment:
I suspect the error before was due to the fact that BigDecimal hasn't gcd method.

Reply:
And the `gcd(x, 0)` was a little bit trickie way to get the absolute value of x